src/main/java/seedu/rc4hdb/model/ModelManager.java

```diff
@@ -55,15 +57,18 @@ public ModelManager(ReadOnlyResidentBook residentBook, ReadOnlyVenueBook venueBo
         filteredResidents = new FilteredList<>(this.residentBook.getResidentList());
 
         // Set up observable instances
+        this.visibleFields = FXCollections.observableArrayList(ResidentField.LOWERCASE_FIELDS);
+        this.hiddenFields = FXCollections.observableArrayList();
         this.observableVenueList = venueBook.getVenueList();
         if (observableVenueList.isEmpty()) {
             logger.info("No venues found in venue list.");
             this.observableBookingList = FXCollections.observableArrayList();
+            this.currentlyDisplayedVenueName = new ObservableItem<>(null);
```

Comment:
Probably could use an optional object just to reduce the risk of Null Pointer exceptions.